hal_st/stm32fxxx/UartStm.hpp

```diff
@@ -8,7 +8,7 @@
 
 namespace hal
 {
-#if defined(STM32WB)
+#if defined(STM32WB) | defined(STM32WBA)
```

Comment:
can this one rely on `HAS_PERIPHERAL_LPUART`?

Reply:
Yeap, it's possible. I've already test it. Changed.